tests/mechanic/provisioner_test.py

```diff
@@ -18,12 +18,15 @@
 import os
 import tempfile
 import unittest.mock as mock
+from pathlib import Path
 from unittest import TestCase
 
 from esrally.mechanic import provisioner, team
 from esrally.utils import convert
 from esrally import exceptions
 
+HOME_DIR = str(Path.home())
```

Comment:
While the `Path` module is really great with its object oriented interface, in Rally we strive for consistency and typically use `os.path.expanduser("~")` for grabbing home. Example: https://github.com/elastic/rally/blob/cc689dfdd30dca5626847cf939018992e9745e58/esrally/log.py#L42